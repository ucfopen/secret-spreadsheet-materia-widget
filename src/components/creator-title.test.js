import React from 'react';
import Title from './creator-title';
import renderer from 'react-test-renderer';

describe('CreatorTitle component', function() {

  beforeEach(() => {
		jest.resetModules()
  })

  test('CreatorTitle renders title', () => {
		const props = {
      title: 'mock-title',
      showIntro: jest.fn(),
      editTitle: jest.fn(),
      onChange: jest.fn(),
      onBlur: jest.fn(),
    }

		const component = renderer.create(<Title {... props}/>);
		let tree = component.toJSON();
    expect(tree).toMatchSnapshot();

  })

})
