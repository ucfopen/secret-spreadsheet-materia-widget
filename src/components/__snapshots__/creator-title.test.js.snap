// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreatorTitle component CreatorTitle renders title 1`] = `
<div
  className="title-bar"
>
  <span
    className="title-text"
  >
    mock-title
  </span>
  <button
    className="edit-title-button"
    onClick={[MockFunction]}
  >
    Edit title
  </button>
  <button
    className="help-button"
    onClick={[MockFunction]}
    type="button"
    value="Help"
  >
    <img
      src="./assets/img/question-mark.png"
    />
    Help
  </button>
</div>
`;
